versioning/requirements.txt

```diff
@@ -1 +1,3 @@
+github_release
+requests
```

Comment:
github_release already installs requests as a dependency and we don't explicitly use it in this code, so this might not be necessary.

Reply:
true

Reply:
will remove